lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -408,3 +408,36 @@ def test_mode_change_after_refund_seat(self, course_modes, new_mode):
 
         assert refund_success
         self.assertEqual(enrollment.mode, new_mode)
+
+    @httpretty.activate
+    @patch('slumber.Resource.put', side_effect=Exception())
+    @patch('lms.djangoapps.commerce.utils.log.info')
+    def test_refund_failure_notification_is_not_sent_if_disabled(self, mocked_log_info, mocked_put):
+        """
+        Test that refund failure notification is not sent if feature flag DISABLE_REFUND_FAILURE_NOTIFICATION
+        is set to True.
+        """
+        features = settings.FEATURES
+        features['DISABLE_REFUND_FAILURE_NOTIFICATION'] = True
+        with override_settings(FEATURES=features):
+            course_id = CourseLocator('test_org', 'test_course_number', 'test_run')
+            CourseMode.objects.all().delete()
+            course_mode = 'verified'
+            CourseModeFactory.create(
+                course_id=course_id,
+                mode_slug=course_mode,
+                mode_display_name=course_mode,
+            )
+            httpretty.register_uri(
+                httpretty.POST,
+                settings.ECOMMERCE_API_URL + 'refunds/',
+                status=201,
+                body='[1]',
```

Comment:
Why are we using these values?

Reply:
No particular reason. Just replicated the test cases above as needed.